### DISCUSSION TITLE
Should the website source live in a separate repository?

### DISCUSSION THREAD

**Discussion by binyamin** [go-task/task#574] [Ideas]
It might be easier to locate it. I can't think of a good reason to include not-yet-built documentation beside the  CLI logic.

**Comment by andreynering** [comment]
Hi @binyamin,

There are good reasons keep the documentation in the same repository.

In short: It makes it easier to people to contribute to new features, and for me to review them: code and documentation changes can be done on the same PR, without having to deal with two different repos. This lower the barrier to new contributors IMO.
